packages/bundles/kairos-overlay-files/collection.yaml

```diff
@@ -0,0 +1,8 @@
+packages:
+  - name: "kairos-overlay-files"
+    category: "bundles"
+    version: "2.4.0"
+    labels:
+      github.repo: "kairos"
+      github.owner: "kairos-io"
+      autobump.strategy: "release"
```

Comment:
ah, this wont work rigth? It needs kairos to be bumped and then this will be bumped but that means when we bump kairos to 2.5.0 this will get bumped to the same version BUT the package installed on kairos 2.5.0 wont be that one....

I would not let it autobump, especially with no changes.

Reply:
You are right. I had no idea what this did. I just copied from the [other package](https://github.com/kairos-io/packages/tree/main/packages/bundles/kairos-default-configs). Btw that package is just a subset of this new one. I wonder if that is being used at all. Maybe we should remove it?

Reply:
I dont hink tha tits used anywhere but at least the bumping works...